marathon_acme/tests/fake_marathon.py

```diff
@@ -20,28 +21,31 @@ def __init__(self):
         self._apps = {}
         self.event_callbacks = []
 
-    def add_app(self, app):
+    def add_app(self, app, client_ip=None):
         # Store the app
         app_id = app['id']
         assert app_id not in self._apps
         self._apps[app_id] = app
 
+        self.trigger_event('api_post_event',
+                           clientIp=client_ip,
+                           uri='/v2/apps/' + app_id.lstrip('/'),
+                           appDefinition=app)
+
     def get_apps(self):
         return list(self._apps.values())
 
-    def attach_event_stream(self, callback, remote_address=None):
+    def attach_event_stream(self, callback, client_ip=None):
         assert callback not in self.event_callbacks
 
         self.event_callbacks.append(callback)
-        self.trigger_event('event_stream_attached',
-                           remoteAddress=remote_address)
+        self.trigger_event('event_stream_attached', clientIp=client_ip)
```

Comment:
The Marathon docs are confusing.. they call the field `clientIp`. But when I test our deployed Marathon (Seed QA and QA) the field is `remoteAddress` for `event_stream_attached`/`event_stream_detached`. The Marathon code seems to confirm this.

But it's definitely `clientIp` for the `api_post_event`.